.gitignore

```diff
@@ -39,8 +39,10 @@ webportal/webapp/.DS_Store
 webportal/webapp/Thumbs.db
 
 # Backend files
-webportal/lambda/__pycache__
-webportal/lambda/.chalice
```

Comment:
As it turns out, it is SUPER dangerous to ignore .chalice... there are deployments that are kept track of by Chalice that live in this folder (as well as the actual chalice configs). There was a previous version of these configs in this repo which is how deployments still work and don't create new Gateways everytime.